src/store/modules/SecurityAndAccess/UserManagementStore.js

```diff
@@ -128,8 +129,8 @@ const UserManagementStore = {
 
           const errorMsg = error.response?.data?.error?.code;
 
-          switch (errorMsg) {
-            case 'Base.1.8.1.PropertyValueFormatError':
+          switch (true) {
```

Comment:
why true?

Reply:
I understand that adding true will allow us to condition checks in case unlike using string. This change makes sense.